saleor/graphql/order/tests/test_order.py

```diff
@@ -3430,11 +3456,25 @@ def test_order_lines_create_with_existing_variant(
     data = content["data"]["orderLinesCreate"]
     assert data["orderLines"][0]["productSku"] == variant.sku
     assert data["orderLines"][0]["quantity"] == old_quantity + quantity
+    if status == OrderStatus.DRAFT:
```

Comment:
The same thing like in logic, we could have function for assertion.

Reply:
fixed it